README.md

```diff
@@ -1,77 +1,53 @@
+* Running as a daemon
+* Listening on the loopback address to send all outgoing DNS queries received on that address out over TLS
+* Using a default configuration which provides Strict Privacy and uses a subset
+of the available [DNS Privacy servers](https://dnsprivacy.org/wiki/x/E4AT)
 
-Stubby is available in the 1.1 release of getdns.  
+# Documentation
 
-# Installing Using a Package Manager
+<!---
+Where should the 'official' Stubby homepage be?
+-->
+See [Stubby Homepage](https://getdnsapi.net/blog/dns-privacy-daemon-stubby/) for more details
 
-Check to see if the 1.1 release of getdns is available via a package manager. Details on Supported Platforms can be found in the 'Supported Platforms' section of the [README](https://getdnsapi.net/documentation/readme)  documentation. 
+# Dependancies
 
-Note: a Homebrew package for Stubby is on the way. 
+Stubby uses [getdns](https://getdnsapi.net/) and requires the 1.1.2 release of getdns or later.
 
-# Building Stubby from Source
+# Installing Using a Package Manager
 
-## Dependencies
+Check to see if getdns and Stubby are available via a package manager.
 
-For Stubby, the only dependency is OpenSSL (version 1.0.2 or later is required for hostname authentication to be supported). If OpenSSL is installed in a non-standard location on your system use the `--with-ssl` option to `configure` below to specify where it is installed.
+Note: a Homebrew package for getdns exists, one for Stubby is on the way.
 
-### Linux
+If you need to install getdns from source, see the section [at the end of this document.](#building-getdns-from-source)
 
-It may be necessary to install [1.0.2 from source](https://openssl.org/source/openssl-1.0.2h.tar.gz) for most Linux distros.
+# Build Stubby from source
 
-### OS X
-
-It is recommended to [install OpenSSL using homebrew](http://brewformulas.org/Openssl), in which case use the following in the `configure` line in the build step below:
-
-```sh
---with-ssl=/usr/local/opt/openssl/
+Get the code:
 ```
-
-## Download the getdns source
-
-Either clone the code:
-
-```sh
-> git clone https://github.com/getdnsapi/getdns.git
-> cd getdns
-> git checkout develop
+git clone https://github.com/getdnsapi/stubby.git
 ```
-for the very latest version of getdns or grab a release tarball from this page: [Latest getdns releases](https://getdnsapi.net/releases/)
 
-## Build the code
-
-Note that on Mac OS X you will need the developer tools from Xcode to compile the code. And you may need to use brew to install libtool (and then use glibtoolize below), autoconf and automake.
-
-```sh
-> git submodule update --init
-> libtoolize -ci
-> autoreconf -fi
-> mkdir build
-> cd build
-> ../configure --prefix=<install_location> --without-libidn --enable-stub-only --enable-debug-daemon
-> make
-> sudo make install
+Build and install (assuming the getdns is installed in a standard location e.g. by homebrew)
```

Comment:
Yes e.g. stubby gets installed in a homebrew cellar and then linked to /usr/local/bin/stubby